Doc/library/sqlite3.rst

```diff
@@ -423,21 +423,20 @@ Connection Objects
 
    .. method:: commit()
 
-      This method commits the current transaction. If you don't call this method,
-      anything you did since the last call to ``commit()`` is not visible from
-      other database connections. If you wonder why you don't see the data you've
-      written to the database, please check you didn't forget to call this method.
```

Comment:
IMO, the original is way too verbose. I'm not sure we need the tongue-in-cheek parts on basic transaction handling. Can we assume that a person using the sqlite3 module has a minimum understanding of basic database concepts, such as transactions? I believe so. Maybe I'm wrong.

Reply:
Seems reasonable to me 👍

Reply:
Agreed—this is a reference document, not a tutorial, so it should be concise and to the point.

Reply:
Thanks both. I'll mark this as resolved.